README.md

```diff
@@ -315,6 +315,54 @@ to customize those behaviours, these methods can be overridden when the
 application defines its own session service that extends the one provided by
 Ember Simple Auth.
 
+You can create your own session service that extends SimpleAuthSessionService in order to customize the `handleAuthentication` method to
+redirect to a default route or possibly load the current user.
+```
+import SimpleAuthSessionService from 'ember-simple-auth/services/session';
+import { inject as service } from '@ember/service';
+
+export default class SessionService extends SimpleAuthSessionService {
+    /**
+     * Inject the router service
+     *
+     * @var {Service}
+     */
+    @service router;
+
+    /**
+     * Inject the current user service
+     *
+     * @var {Service}
+     */
+    @service currentUser;
+
+    /**
+     * Overwrite the handle authentication method
+     *
+     * @var {Service}
+     */
+    handleAuthentication() {
+        this.router.transitionTo('dashboard.boards');
```

Comment:
I would write `super.handleAuthentication('dashboard.boards')` instead. That way, the default logic defined [here](https://github.com/simplabs/ember-simple-auth/blob/e93a120a187943cb2453976f81912f9b8ef83247/packages/ember-simple-auth/addon/-internals/routing.js#L33-L50) is kept as well.

Reply:
@bertdeblock How would I pass `owner` if I did that ? 
```js
export function handleSessionAuthenticated(owner, routeAfterAuthentication) {
```

Reply:
https://api.emberjs.com/ember/release/functions/@ember%2Fapplication/getOwner